tests/use/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -29,6 +39,14 @@ use MyVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeyLon
 use some\\Full\\{ClassA, ClassB, ClassC as C};
 use function some\\Full\\{fn_a, fn_b, fn_c};
 use const some\\Full\\{ConstA, ConstB, ConstC};
+use Mizo\\Web\\{Php\\WebSite,
+    Php\\KeyWord,
+    Php\\UnicodePrint,
+    JS\\JavaScript,
+    function JS\\printTotal,
+    function JS\\printList,
+    const JS\\BUAIKUM,
+    const JS\\MAUTAM};
```

Comment:
i think this might have gotten solved in the other PR, but line break for these braces

Reply:
@mgrip yep, already solved, need rebase :+1: